domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/casetype/CaseTypeService.java

```diff
@@ -7,10 +7,11 @@
 import java.util.Collection;
 import java.util.List;
 import java.util.Optional;
+import java.util.Set;
 
 public interface CaseTypeService {
 
-    void createAll(JurisdictionEntity jurisdiction, Collection<CaseTypeEntity> caseTypes);
+    void createAll(JurisdictionEntity jurisdiction, Collection<CaseTypeEntity> caseTypes, Set<String> missingUserRoles);
```

Comment:
minor feel free to ignore. We could make this method a bit more reusable by passing the whole parse context? This way we can pass other type of info in the future down to the CaseTypeServiceImpl without api changes.